package-lock.json

```diff
@@ -3973,7 +3971,7 @@
         "original-require": "1.0.1",
         "require-from-string": "2.0.2",
         "simple-git": "^1.107.0",
-        "snarkjs": "github:LimeChain/snarkjs",
+        "snarkjs": "github:LimeChain/snarkjs#f7036ebe9bfa3f3817077f050780c8ec1542e249",
```

Comment:
nor this. I wonder if it depends on the version of `npm` you're using at the time? I think this change is more correct though since it locks it to a specific commit rather than just whatever is on `LimeChain/snarkjs`.